target/linux/ramips/image/mt7621.mk

```diff
@@ -273,10 +272,10 @@ define Device/dlink_dir-xx60-a1
   DEVICE_VENDOR := D-Link
   DEVICE_PACKAGES := kmod-mt7615e kmod-mt7615-firmware kmod-usb3 \
 	kmod-usb-ledtrig-usbport
-  KERNEL := $$(KERNEL) | uimage-padhdr 96
+  KERNEL := $$(KERNEL) | uimage-sgehdr
   IMAGES += factory.bin
   IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
-  IMAGE/factory.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-ubi | \
+  IMAGE/factory.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-ubi |\
```

Comment:
This change seems unneeded and unrelated.

Reply:
was the only "Whitespace" thing i could find

Reply:
So this needs to become `IMAGE/factory.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-ubi | \`

Reply:
So with a space between `|` and `\`.

Reply:
Yes, since this would mean it's _not changed_.

Reply:
add space